assets/src/adaptivity/scripting.ts

```diff
@@ -177,7 +177,9 @@ export const applyState = (
               } else {
                 // it's not supposed to be a string however, so we need to coerce it
                 const coerced = coerceCapiValue(newValue, targetType);
-                script += `= ${coerced};`;
+                // need to wrap the content in "" as it's a string else it will be evaluated as undefined as the script
+                //generates the expression as 'let {stage.something.value} = Finches'
+                script += `= "${coerced}";`;
```

Comment:
since in this branch it's not supposed to be a string, my guess is that maybe its an ENUM? I think we should check
if the type of `coerced` is a string *then* wrap it in quotes, we wouldn't want to wrap numbers at this point